### ISSUE TITLE
About starting

### ISSUE THREAD

**Issue by Yuu2311** [vivinano/MudaeAutoBot#72]
when I start the bot, there are so many lines keep printing, do I need to wait this for a while or is it already started ?

**Comment by vivinano** [comment]
if its printing alot of lines then you have the wrong version of discum 

bot requires discum version 1.3+ 

https://github.com/Merubokkusu/Discord-S.C.U.M

please refer here to install the new version

**Comment by Yuu2311** [comment]
Sorry because I have 0 experience in Python programming and don't know how to install that. Could you help me?

**Comment by vivinano** [comment]
No need to apologize. basically since `Discum` is pypi version is 1.1 most people get 1.1 and think they are good to go but the latest version is actually on their github repo. 

I've attempted to solve issues in terms of `Discum` Installation myself but sometimes it isnt as simple as saying run:
 `python -m pip install --user --upgrade git+https://github.com/Merubokkusu/Discord-S.C.U.M.git#egg=discum`
in your `Command Prompt` 

The reason I say go to `https://github.com/Merubokkusu/Discord-S.C.U.M` is that they are more knowledgeable about `Discum`

Basically I'm the guy who can help with `MudaeAutoBot` but over at that link above they are the guys that made `Discum` and are more knowledgeable on that subject

You just do the same thing you did on this section you can create an issue over on their page and one of them will answer they are really cool people. I email them all the time and they even will add you on discord and help on a 1 on 1 basis and all of that.

**Comment by Yuu2311** [comment]
so I have already installed everything and run the program, but this shows
![image](https://user-images.githubusercontent.com/86243871/125195833-b0373a00-e281-11eb-8ad3-6b41d1f8f9b4.png)
what i need to do next?

**Comment by vivinano** [comment]
it should be good to go sometimes it errors if mudae replies with something that doesnt have an embed but it shouldnt effect the running status

try a test snipe case like change the values of min_kak to like 10 and have it snipe a character

**Comment by Yuu2311** [comment]
oh I can use it now, thank you for your help XD

**Comment by vivinano** [comment]
no problem I will be closing this issue 

Please Star this repo if you want to make sure the repo gains more popularity thank you
